src/components/Give/index.js

```diff
@@ -127,17 +127,16 @@ class App extends Component {
     let {
       donationAmount,
       accounts,
-      activeDonateModal,
```

Comment:
@jschiarizzi so we don't want to use activeDonateModal cause that doesn't pass in like mutt #1, it passes in 0 for instance for the very first dog on the list. now this is the actual mutt #